core/trino-main/src/main/java/io/trino/operator/MultiChannelGroupByHash.java

```diff
@@ -980,55 +990,54 @@ public GroupByIdBlock getResult()
         }
     }
 
-    private abstract class LowCardinalityDictionaryWork<T>
-            implements Work<T>
+    /**
+     * Returns an array containing a single Page position occurrence that corresponds to the
+     * low cardinality dictionary combinationId at that index, or a value of -1 if no position
```

Comment:
Simplify
`to the low cardinality dictionary combinationId at that index, or` -> `to the low dictionary combinationId, or`

index = combinationId, no need to distinguish between these two